tests/python/rest_api/test_jobs.py

```diff
@@ -728,6 +745,7 @@ def _get_or_create_gt_job(self, user, task_id, frames):
         with make_api_client(user) as api_client:
             (task_jobs, _) = api_client.jobs_api.list(task_id=task_id, type="ground_truth")
             if task_jobs.results:
+                assert not frames, "specific frames requested but gt_job exists"
```

Comment:
It makes sense to split the `self._get_or_create_gt_job` function in 2 now to exclude the possibility of incorrect invocation. Please split, update the code, and this PR seems to be finished.